src/components/Select/SelectT.tsx

```diff
@@ -44,7 +45,7 @@ export default function SelectT<T>(props: SelectTProps<T>): JSX.Element {
     placeholder,
     errorText,
     warningText,
-    readonly = true,
+    readonly,
```

Comment:
What's the difference between readonly and disabled in this component?

Reply:
Styles and also readonly allows focus while disabled not

Reply:
Do we have a use-case for readonly? This component is rather complex - will be nice to remove any semantics that aren't needed.

Reply:
Maybe later

Reply:
We don't. Chudo said in slack: "Read-only is used in cases where it may be permission related or maybe one field is read-only because it is dependent on another field's value. We may not need it at all yet in our application."

Reply:
Chudo's reference I believe was a bit generic on nomenclature - not sure he was giving us a requirement.
Will be cleaner and less confusing to remove props that aren't needed? I can remove it if we agree it's ok to remove it.

Reply:
I can follow up with Chudo, ignore my comment for now.